buildenv/jenkins/aqaTestPipeline.groovy

```diff
@@ -6,10 +6,16 @@ def TARGETS = params.TARGETS ?: "Grinder"
 TARGETS = TARGETS.trim().split("\\s*,\\s*")
 
 def PARALLEL = params.PARALLEL ? params.PARALLEL : "Dynamic"
-def NUM_MACHINES = params.NUM_MACHINES ? params.NUM_MACHINES : 3
+
+def NUM_MACHINES = ""
+if (params.NUM_MACHINES) {
+    NUM_MACHINES = params.NUM_MACHINES
+} else if (!params.TEST_TIME) {
+    NUM_MACHINES = 3
```

Comment:
Can we document this magic number (why 3?)

Reply:
This is a default value if params.NUM_MACHINES is not set